task/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>

```

Comment:
Add a proper title for the HTML file

Reply:
yes @Sachinrawat1215 i solve this issue and i am working on it. Thankyou